cypress/integration/transfers/native/transfer-bitcoin/test.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-undef */
+/*
+    End 2 End testing
+ */
+
+const ASSET = 'BTC'
+const TITLE = ' Test ' + ASSET + ' transfer'
+const NO_BROADCAST = true
+
+let TEST_SEED = Cypress.env('TEST_SEED')
+let TEST_SEED_LABEL = Cypress.env('TEST_SEED_LABEL')
+if (!TEST_SEED) throw Error('TEST ENV INVALID! required: TEST_SEED')
+if (!TEST_SEED_LABEL) throw Error('TEST ENV INVALID! required: TEST_SEED_LABEL')
+
+import {sleep} from 'wait-promise';
```

Comment:
don't import a whole package for this, write or grab a one liner `await sleep(ms: number)` util from stack overflow

even better, don't sleep, await some DOM change as it's non deterministic and inefficient